packages/database/src/migrations/20210127064624-MedicalCertificateLegendColorChange-modifies-data.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+import { updateValues } from '../utilities/migration';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const dashboardReportId = 'TO_CD_Medical_Certs_Distributed';
+const previousLabels = {
+  CD73: 'Missionary - LDS',
+  CD74: 'Missionary - Other',
+  CD75: 'Visa - Tonga Immigration',
+  CD76: 'Visa - China',
+  CD77: 'Visa - Fiji',
+  CD81: 'Visa - Malaysia',
+  CD82: 'Visa - Samoa (Apia and Pangopango)',
+  CD83: 'Visa - Thailand',
+  CD84: 'Visa - Other',
+  CD85: 'Employment - Bankers (BSP, TDB)',
+  CD86: 'Employment - FWC Workers',
+  CD87: 'Employment - South Pacific Business Development (SPBD)',
+  CD88: 'Employment - Seafarers',
+  CD89: 'Employment - Tonga Civil Servant (all government workers)',
+  CD90: 'Employment - Other',
+  CD84a: 'Employment - Army, Firemen, Wardens',
+};
+const newLabels = {
+  CD73: 'LDS',
+  CD74: 'other',
```

Comment:
We got three difference columns names `Other - ${option name}` and we need them to be `Other` with different colors. But it seems like the front end didn't support three same key and it will combine them into one column. So I rename them as `Other`, `other`, `others`. It looks a bit weird but do you suggest we make another card to support it and keep it for now?

Reply:
I see! It is not ideal, but if the product owner is fine with those labels, I'm also fine. Please do create a card for this though!

Reply:
Will do after merge